### PR TITLE
deploy: move compatibility changes to a more noticeable position in upgrade guide

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Move compatibility changes to a more noticeable position in upgrade guide

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: upgrade-tidb-using-tiup.md [hunk 1]
```diff
@@ -136,7 +140,7 @@ Now, the offline mirror has been upgraded successfully. If an error occurs durin
 >
 > Before you upgrade the cluster to v6.5.0, make sure that the parameters you have modified in v4.0 are compatible in v6.5.0. For details, see [TiKV Configuration File](/tikv-configuration-file.md).
 
-### Step 3: Check the health status of the current cluster
+### Step 5: Check the health status of the current cluster
```
Comment (shichun-0415) [inline upgrade-tidb-using-tiup.md:143]:
```suggestion
### Step 4: Check the health status of the current cluster
```

Comment (shichun-0415) [reply upgrade-tidb-using-tiup.md:143]: Change [Line 158](https://github.com/pingcap/docs/pull/12408/files#diff-e210c3a991b4061481b908eed6924a0b5c8acfa7528d7c731b3308321f811a97R158) to Step 5.

Comment (ran-huang) [reply upgrade-tidb-using-tiup.md]: Done. Thanks

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**